meta-lmp-base/recipes-support/bluetooth-attach/bluetooth-attach/hciattach-custom.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
```
set -e
```
can be usefull to avoid errors in the midle

Reply:
> ```
> set -e
> ```
> 
> can be usefull to avoid errors in the midle

@quaresmajose not sure if this is an option for this case.

Reply:
```
hciconfig hci0 up && echo OK || echo FAIL
```
you can relay on this last command but you will need to return something to the caller because echo allway return true.so you need to save the return code of the hciconfig to return

Reply:
this is how it's done in the test scripts:
```
# Return the exit code of the first command when using pipe.
pipe0_status() {
    [ "$#" -ne 2 ] && error_msg "Usage: pipe0_status cmd1 cmd2"
    # shellcheck disable=SC2039
    local cmd1="$1"
    # shellcheck disable=SC2039
    local cmd2="$2"

    exec 4>&1
    # shellcheck disable=SC2039
    local ret_val
    ret_val=$({ { eval "${cmd1}" 3>&-; echo "$?" 1>&3; } 4>&- \
              | eval "${cmd2}" 1>&4; } 3>&1)
    exec 4>&-

    return "${ret_val}"
}

pipe0_status "./runltp -p -q -f shardfile \
                                 -l ${OUTPUT}/LTP_${LOG_FILE}.log \
                                 -C ${OUTPUT}/LTP_${LOG_FILE}.failed \
                                 -d ${LTP_TMPDIR} \
                                    ${SKIPFILE}" "tee ${OUTPUT}/LTP_${LOG_FILE}.out"
```
Since `tee` is like `echo` it would always return true. The tests fail when the exit code is not 0 so the pipe0 saves the `runltp` exit code.